src/ert/run_models/base_run_model.py

```diff
@@ -90,8 +101,9 @@ def __init__(
         self._job_start_time: int = 0
         self._job_stop_time: int = 0
         self._indeterminate: bool = False
-        self._fail_message: str = ""
-        self._failed: bool = False
+        self._failed = False
+        self._exception: Optional[Exception] = None
+        self._messages: MutableSequence[str] = []
```

Comment:
It looks like these `_messages` are always "error messages". If so, should we rename the variable accordingly?